src/types.ts

```diff
@@ -830,80 +893,258 @@ export interface LiveActivityRequest {
+  | LiveActivityDismissalPolicyImmediate
+  | LiveActivityDismissalPolicyDefault
+  | LiveActivityDismissalPolicyAfterDate;
 
 /**
  * Dismissal policy to immediately dismiss the Live Activity on end.
  */
 export interface LiveActivityDismissalPolicyImmediate {
-  type: "immediate"
+  type: 'immediate';
 }
 
 /**
  * Dismissal policy to dismiss the Live Activity after the expiration.
  */
 export interface LiveActivityDismissalPolicyDefault {
-  type: "default"
+  type: 'default';
 }
 
 /**
  * Dismissal policy to dismiss the Live Activity after a given date.
  */
 export interface LiveActivityDismissalPolicyAfterDate {
-  type: "after"
-  // ISO 8601 date string
-  date: string
-}
\ No newline at end of file
+  type: 'after';
+  // ISO 8601 date string.
+  date: string;
+}
+
+/**
+ * Live Update info.
+ */
+export interface LiveUpdate {
+  /**
+   * The Live Update name.
+   */
+  name: string;
+
+  /**
+   * The Live Update type.
+   */
+  type: string;
+
+  /**
+   * Dynamic content.
+   */
+  content: JsonObject;
+
+  /**
+   *  ISO 8601 date string of the last content update.
+   */
+  lastContentUpdateTimestamp: string;
+
+  /**
+   * ISO 8601 date string of the last state update.
+   */
+  lastStateChangeTimestamp: string;
+
+  /**
+   * Optional ISO 8601 date string that defines when to end this Live Update.
+   */
+  dismissTimestamp?: string;
+}
+
+
+/**
+ * Live Update list request.
+ */
+export interface LiveUpdateListRequest {
+  type: string;
+}
+
+/**
+ * Live Update update request.
+ */
+export interface LiveUpdateUpdateRequest {
+  /**
+   * The Live Update name.
+   */
+  name: string;
+  /**
+   * Dynamic content.
+   */
+  content: JsonObject;
+
+  /**
+   * Optional ISO 8601 date string, used to filter out of order updates/
+   */
+  timestamp?: string;
+
+  /**
+   * Optional ISO 8601 date string that defines when to end this Live Update.
+   */
+  dismissTimestamp?: string;
+}
+
+/**
+ * Live Update end request.
+ */
+export interface LiveUpdateEndRequest {
+  /**
+   * The Live Update name.
+   */
+  name: string;
+
+  /**
+   * Dynamic content.
+   */
+  content?: JsonObject;
+
+  /**
+   * Optional ISO 8601 date string, used to filter out of order updates/
+   */
+  timestamp?: string;
+
+  /**
+   * Optional ISO 8601 date string that defines when to end this Live Update.
+   */
+  dismissTimestamp?: string;
+}
+
+/**
+ * Live Update create request.
+ */
+export interface LiveUpdateCreateRequest {
+  /**
+   * The Live Update name.
+   */
+  name: string;
+
+  /**
+   * The Live Update type.
+   */
+  type: string;
+
+  /**
+   * Dynamic content.
+   */
+  content: JsonObject;
+
+  /**
+   * Optional ISO 8601 date string, used to filter out of order updates/
+   */
+  timestamp?: string;
+
+  /**
+   * Optional ISO 8601 date string that defines when to end this Live Update.
+   */
+  dismissTimestamp?: string;
+}
+
+/**
+ * Live Update manager.
+ */
+export class AirshipLiveUpdateManager {
```

Comment:
didn't this get pulled out into it's own file? not sure if I'm missing something there